src/services/TransactionRequestService.ts

```diff
@@ -44,7 +44,7 @@ export class TransactionRequestService {
             if (bill.creditor.publickey === addr) {
                 const changes = balanceChanges[addr];
                 for (const change of changes) {
-                    if (change.value === "XRP" && Number(change.value) * 1000000 === tr.totalXrp) {
+                    if (change.value === "XRP" && Number(change.value) * 1000000 === tr.totalXrpDrops) {
```

Comment:
I think there's a bug here, not related to your changes tho.
I think it should say change.currency === "XRP"

Reply:
Was indeed a bug, resolved this and changed the code to use XRPUtil